index.d.ts

```diff
@@ -1,46 +1,66 @@
-// TypeScript Version: 3.7
+// TypeScript Version: 3.3
```

Comment:
I set this down to the lowest version that passed dtslint

Reply:
what's the command for this! this would be super useful :)